src/mii_sequences/elias_fano.rs

```diff
@@ -571,23 +571,31 @@ impl EliasFanoBuilder {
     /// - `universe`: The (exclusive) upper bound of integers to be stored, i.e., an integer in `[0..universe - 1]`.
     /// - `num_vals`: The number of integers that will be pushed (> 0).
     ///
-    /// # Errors
-    ///
-    /// An error is returned if `num_vals == 0`.
-    pub fn new(universe: usize, num_vals: usize) -> Result<Self> {
+    pub fn new(universe: u64, num_vals: usize) -> Self {
```

Comment:
Could you not remove this error handling? `EliasFano`'s behavior is undefined when given empty input. For example, an `EliasFano` instance created by `EliasFanoBuilder::build()` with `num_vals==0` cannot be guaranteed to function correctly.